### PR TITLE
Add king_taco

### LINKED ISSUE #9820: King Taco
### Brand name

King Taco

American fast food restaurant chain

### Wikidata ID

Q6412104
https://www.wikidata.org/wiki/Q6412104
https://www.wikidata.org/wiki/Special:EntityData/Q6412104.json

### Store finder url(s)

Official Url(s): https://kingtaco.com/
- [ ] I have run the below and if anything in generated, added it.
- [ ] I have manually checked for a storefinder
- [ ] This is not a manufacturer listing outlets operating under their own brands / this will not create duplicates
`pipenv run scrapy links https://kingtaco.com/`
`pipenv run scrapy sitemap --pages https://kingtaco.com/`
`pipenv run scrapy sf --brand-wikidata=Q6412104 https://kingtaco.com/`
pipenv run scrapy sf https://kingtaco.com/locations/
```
from locations.storefinders.maps_marker_pro import MapsMarkerProSpider


class KingTacoSpider(MapsMarkerProSpider):
    name = "king_taco"
    item_attributes = {
        "brand_wikidata": "Q6412104",
        "brand": "King Taco",
    }
    allowed_domains = [
        "kingtaco.com",
    ]
from locations.storefinders.wp_store_locator import WPStoreLocatorSpider


class KingTacoSpider(WPStoreLocatorSpider):
    name = "king_taco"
    item_attributes = {
        "brand_wikidata": "Q6412104",
        "brand": "King Taco",
    }
    allowed_domains = [
        "kingtaco.com",
    ]

```

### PR DESCRIPTION
**Pull request by CloCkWeRX**
Fix #9820

### CODE REVIEW

**Review by davidhicks — CHANGES_REQUESTED** [review]
Suggestions for bringing this spider up to the proposed standard set in #9916:

* Rename spider to have "_us" suffix.
* Specify `days=DAYS_EN`.
* Remove `time_format`.
* Add `"extras": Categories.FAST_FOOD.value` to `item_attributes`.

**Comment by CloCkWeRX** [comment]
time_format is still needed at the moment

**Review by davidhicks — CHANGES_REQUESTED** [review]
Now that #9916 is merged, the final change needed here is to remove the `time_format` attribute as this is deprecated/no longer required. Then this is good to go.

File: locations/spiders/king_taco_us.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+from locations.storefinders.wp_store_locator import WPStoreLocatorSpider
+
+
+class KingTacoUSSpider(WPStoreLocatorSpider):
+    name = "king_taco_us"
+    item_attributes = {"brand_wikidata": "Q6412104", "brand": "King Taco", "extras": Categories.FAST_FOOD.value}
+    allowed_domains = [
+        "kingtaco.com",
+    ]
+    time_format = "%I:%M %p"
```
Comment (CloCkWeRX) [inline locations/spiders/king_taco_us.py:12]:
```suggestion
```

**Review by davidhicks — APPROVED** [review]

**Pull request merged**